src/de.rs

```diff
@@ -348,6 +348,57 @@ where
     Ok(deserializer.deserialize_map(visitor)?)
 }
 
+/// Interpret a [`Feature`] as an instance of type `T`.
+///
+/// This is analogous to [`serde_json::from_value`](https://docs.rs/serde_json/latest/serde_json/fn.from_value.html)
+///
+/// `T`'s `geometry` field will be deserialized from `feature.geometry`.
+/// All other fields will be deserialized from `feature.properties`.
+///
+/// # Examples
+#[cfg_attr(feature = "geo-types", doc = "```")]
```

Comment:
Have we done this before? I love it.

Reply:
I've been known to sprinkle it around. It's pretty arcane, but it does the trick. I was kind of shocked the first time I tried it and it worked.